src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeedFormatter.cs

```diff
@@ -2,191 +2,169 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Runtime.Serialization;
+using System.Xml;
+
 namespace System.ServiceModel.Syndication
 {
-    using System;
-    using System.Diagnostics;
-    using System.Globalization;
-    using System.Runtime.Serialization;
-    using System.Xml;
-    using DiagnosticUtility = System.ServiceModel.DiagnosticUtility;
-
     public delegate bool TryParseDateTimeCallback(XmlDateTimeData data, out DateTimeOffset dateTimeOffset);
     public delegate bool TryParseUriCallback(XmlUriData data, out Uri uri);
 
     [DataContract]
     public abstract class SyndicationFeedFormatter
     {
-        private SyndicationFeed _feed;
-
         protected SyndicationFeedFormatter()
         {
-            _feed = null;
+            Feed = null;
             DateTimeParser = GetDefaultDateTimeParser();
         }
 
         protected SyndicationFeedFormatter(SyndicationFeed feedToWrite)
         {
-            if (feedToWrite == null)
-            {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("feedToWrite");
-            }
-            _feed = feedToWrite;
+            Feed = feedToWrite ?? throw new ArgumentNullException(nameof(feedToWrite));
             DateTimeParser = GetDefaultDateTimeParser();
         }
 
-        public SyndicationFeed Feed
-        {
-            get
-            {
-                return _feed;
-            }
-        }
+        public SyndicationFeed Feed { get; private set; }
 
```

Comment:
This will change the result of reflection over this type at runtime. This class is attributed with [DataContract] and the DataContractSerializer does different things if there's a private setter compared with a get only property. Basically this is a breaking change.